staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -147,6 +208,8 @@ func getFinalIP(addrs []net.Addr, family AddressFamily) (net.IP, error) {
 	return nil, nil
 }
 
+// getIPFromInterface gets the IPs on an interface and looks for a global unicast IP. The interface
```

Comment:
changing to say that it "gets the IPs... and returns a global unicast address, if any."